packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -129,6 +131,17 @@ export function isTheia(): boolean {
     return false;
 }
 
+export async function isUsingTokenAuth(profileName: string): Promise<boolean> {
+    const config = (await Profiles.getInstance().getProfileInfo()).getTeamConfig();
+    const profile = await Profiles.getInstance().getLoadedProfConfig(profileName);
+
+    if (profile.type === "zosmf") {
+        return config.properties.profiles["base"].secure.includes("tokenValue");
+    }
```

Comment:
There may be some additional cases we should handle for z/OSMF profiles:
* The token value may not always be stored in a base profile. This is the default behavior of ZE, but it could also be stored in the z/OSMF profile itself.
* The base profile may have a different name other than "base".